.github/workflows/pages.yml

```
@@ -0,0 +1,23 @@
+name: Build Jekyll Pages
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: bom-check
+      uses: arma-actions/bom-check@v1.0
+      with:
+        path: _conferences
+    - name: Set up Ruby 2.6.5
```

```suggestion
    - name: Set up Ruby
```
Just since someone will invariably forget to update both